.github/workflows/msbuild.yml

```diff
@@ -26,6 +26,9 @@ jobs:
     - name: Check code style
       working-directory: dafny/Source
       run: dotnet tool run dotnet-format -w --check Dafny.sln
+    - name: Attempt custom Boogie patch
+      working-directory: dafny
+      run: git apply customBoogie.patch
```

Comment:
Could we also clone Boogie in the expected sibling directly and run `dotnet` build? That will catch more breakages for not too much more effort.

Reply:
One question then is what version of Boogie to clone. I think either a static or the latest Boogie version wouldn't be great, since the former wouldn't detect issues and the latter would fail without changes to the Dafny repository. It should probably be the commit that was used to create the currently referenced version of Boogie, which it a little bit more complicated to setup, but I'll give it a try.